src/components/Button.stories.mdx

```diff
@@ -45,6 +45,7 @@ export const Template = (args) => {
       <Button variant="tertiaryDanger" label="Delete" />
       <Button variant="danger" label="Delete" />
       <Button variant="text" label="Edit" />
+      <Button variant="secondaryBorderless" label="Text" />
```

Comment:
@kbesingeryeh - just curious on the naming of the variant `secondaryBorderless` - Just did some digging around at other design systems and they call this a `text` variant, but I have also seen "flat" or "Ghost" used..

https://ant.design/components/button
https://m3.material.io/components/all-buttons#9134ac95-678e-49ae-a50a-e71948011b05

Reply:
yes, we have a `text` variant already but its a blue text, so I was thinking since this pattern follows the `secondary` button variant, to call it `secondaryBorderless` since its the same just without a border.. I could also see `secondaryText` being a good name too. Thoughts?

Reply:
yeah I like `secondaryText` as it falls in line with the `tertiaryDanger` pattern.